src/vcpkg-test/system.mac.cpp

```diff
@@ -0,0 +1,67 @@
+#include <catch2/catch.hpp>
+
+#include <vcpkg/base/span.h>
+#include <vcpkg/base/stringview.h>
+#include <vcpkg/base/system.mac.h>
+
+using namespace vcpkg;
+
+static constexpr StringLiteral GOOD_ZERO_MAC = "00:00:00:00:00:00";
+static constexpr StringLiteral NON_ZERO_MAC = "90:df:f7:db:45:cc";
+static constexpr StringLiteral ALL_FS_MAC = "ff:ff:ff:ff:ff:ff";
+static constexpr StringLiteral IBRIDGE_MAC = "ac:de:48:00:11:22";
+static constexpr StringLiteral BAD_ZERO_MAC = "00-00-00-00-00-00";
+static constexpr StringLiteral NOT_A_MAC = "00:00:no:jk:00:00";
+static constexpr StringLiteral EMPTY_MAC = "";
+
+TEST_CASE ("validate MAC address format", "[metrics.map]")
```

Comment:
```suggestion
TEST_CASE ("validate MAC address format", "[metrics.mac]")
```
Was this intended to be `metrics.mac`?